package.json

```diff
@@ -109,6 +109,7 @@
     "karma-mocha": "^0.2.2",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
+    "matrix-react-test-utils": "^0.1.1",
```

Comment:
presumably you actually need the next release? doesn't look like we're doing magic to fetch the develop branch of this.